src/xdocs/config_coding.xml

```diff
@@ -3468,6 +3468,22 @@ public class D {
 }
        </source>
        <p>
+         3) If validateOnlyOverlapping is set to false but the variable is locally defined, there
+            will not be a violation provided the variable doesn't overlap, for example:
+       </p>
+       <source>
+public class E {
+  private String s1 = "foo1";
+  String s2 = "foo2";
+  E() {
+  s1 = "bar1"; // Violation. Reference to instance variable 's1' needs "this.".
```

Comment:
Can we indent instructions of the constructor. It is not clear that they are instructions and not just more fields.